### ISSUE TITLE
Yats doesn't work with rainbow

### ISSUE THREAD

**Issue by andrevmatos** [HerringtonDarkholme/yats.vim#132]
You mentioned https://github.com/luochen1990/rainbow here: https://github.com/HerringtonDarkholme/yats.vim/issues/124#issuecomment-512065243
but seems like Yats breaks it. I tried setting different configs at rainbow, but it always seems `typescriptParens` syntax definition always come first and break the plugin.
![image](https://user-images.githubusercontent.com/587021/64737888-405c9a80-d4c4-11e9-86d8-8ef06b01e866.png)

 `typescriptBraces` is also applied to both square and curly brackets, and don't get rainbow'ed.

**Comment by cjnucette** [comment]
I don't think that `rainbow` would be necessary if there were different groups for different braces like in `pangloss/vim-javascript`. That way one could highlight each case differently.

**Comment by HerringtonDarkholme** [comment]
Hmmm, I think remove the highlight of `typescriptBraces` should work.

**Comment by FallenWarrior2k** [comment]
To provide some more context, because I am interested in this myself, Rainbow does apply to named imports, but nothing else in the file.
![image](https://user-images.githubusercontent.com/20320149/79053332-de703900-7c3c-11ea-82b2-a70944444965.png)
